ecosystem/indexer-grpc/indexer-grpc-table-info/src/runtime.rs

```diff
@@ -66,26 +68,47 @@ pub fn bootstrap(
     let db =
         open_db(db_path, &rocksdb_config).expect("Failed to open up indexer async v2 db initially");
 
+    // Set up the gcs bucket
+    let gcs_bucket_name = node_config.indexer_table_info.gcs_bucket_name.clone();
+    let named_chain = match NamedChain::from_chain_id(&chain_id) {
+        Ok(named_chain) => format!("{}", named_chain).to_lowercase(),
```

Comment:
NamedChain is not an accurate way to convert chain id to named chain, e.g., for devnet.

we should consider use external name at deployment side.